src/utils/wallet.js

```diff
@@ -926,6 +986,42 @@ const wallet = {
     storage.setItem(storageKeys.hideZeroBalanceTokens, false);
   },
 
+  /**
+   * Defines if a token should always be shown, despite having zero balance and the "hide zero
+   * balance" setting being active.
+   * @param {string} tokenUid uid of the token to be updated
+   * @param {boolean} [newValue=false] If true, the token will always be shown
+   */
+  setTokenAlwaysShow(tokenUid, newValue = false) {
+    const alwaysShowMap = storage.getItem(storageKeys.alwaysShowTokens) || {};
+    if (!newValue) delete alwaysShowMap[tokenUid];
+    else alwaysShowMap[tokenUid] = true;
```

Comment:
```suggestion
    if (!newValue) {
      delete alwaysShowMap[tokenUid];
    } else {
      alwaysShowMap[tokenUid] = true;
    }
```

Btw, will `storage.getItem` return an object? Don't we have to `JSON.parse` it?

Reply:
Fixed the style on 04b6f48c6579b568b9caf55233c29dca14e31ecb .

Actually, `storage.getItem` is returning an object, and would fail if I tried to `JSON.parse` it. Is this not the expected behaviour?

I even closed the application and server, restarted them again to confirm the persistent storage is holding this data and not only the memory state.